multimedia/libvpx/Portfile

```diff
@@ -72,6 +72,10 @@ configure.args      --enable-vp8 \
                     --disable-examples \
                     --disable-unit-tests
 
+if { ${build_arch} eq "arm64" } {
```

Comment:
This will not give you a correct build when using the universal variant.

Reply:
Thanks for catching this! Hopefully the update I just pushed is the right way to fix this.